src/Test/Unit/Config/Validator/Build/ComposerFileTest.php

```diff
@@ -137,10 +155,27 @@ public function testValidateWrongComposerJson(): void
      */
     public function testValidateMagentoLower23(): void
     {
-        $this->magentoVersionMock->expects($this->once())
+        $this->magentoVersionMock->expects($this->exactly(1))
+            ->method('isGreaterOrEqual')
+            ->withConsecutive(['2.3'])
+            ->willReturnOnConsecutiveCalls(false);
+        $this->fileListMock->expects($this->never())
+            ->method('getMagentoComposer');
+        $this->resultFactoryMock->expects($this->once())
+            ->method('success');
+
+        $this->validator->validate();
+    }
+
+    /**
+     * @throws UndefinedPackageException
+     */
+    public function testValidateMagentoHigherEqual243(): void
+    {
+        $this->magentoVersionMock->expects($this->exactly(1))
             ->method('isGreaterOrEqual')
-            ->with('2.3')
-            ->willReturn(false);
+            ->withConsecutive(['2.3'])
+            ->willReturnOnConsecutiveCalls(false);
```

Comment:
```suggestion
            ->willReturnOnConsecutiveCalls(true, true);
```

Reply:
@billygilbert I have updated the test. If there is anything else please comment as fast as possible. It would be nice if this could be added :). im available till 17:00.